contents/handbook/people/benefits.md

```diff
@@ -24,7 +24,7 @@ In countries where you are employed under Deel's EOR service, we make pension co
 
 We offer private health insurance in countries where the health system struggles with long wait times for appointments or high out-of-pocket costs. Currently, we provide private health insurance in the US and UK.
 
-In the US, our medical insurance is provided via UnitedHealthcare and managed via our payroll provider [Gusto](https://app.gusto.com/). We also offer dental and vision insurance via Guardian. PostHog pays 100% of the premium of the Platinum plan for team members, and 75% for dependents.
+In the US, our medical insurance is provided via UnitedHealthcare and managed via our payroll provider [Deel](https://app.deel.com/). We also offer dental and vision insurance via Guardian. PostHog pays 100% of the premium of the Platinum plan for team members, and 75% for dependents.
```

Comment:
https://github.com/PostHog/posthog.com/blob/b31f19d2e3088c59a7613fcbb07b2e914cfedc1b/contents/teams/people/index.mdx?plain=1#L102 and https://github.com/PostHog/posthog.com/blob/b31f19d2e3088c59a7613fcbb07b2e914cfedc1b/contents/handbook/people/onboarding.md?plain=1#L116 @fraserhopper